src/frontend/README.md

```diff
@@ -44,14 +40,24 @@ else.
 
 ### Metrics
 
-- Automatic Instrumentation (Agent): :large_blue_diamond:
-- Automatic Instrumentation (OTel Operator): :red_circle:
-- Instrumentation Libraries  : :red_circle:
-- Manual Metric Creation:  :red_circle:
-- Collector Metric Transformation::red_circle:
-- Push Metrics: :red_circle:
-- SLO Metrics: :red_circle:
-- Multiple Metric Instruments: :red_circle:
+- [Automatic Instrumentation (OTel
```

Comment:
If this is based on the OTel Operator, maybe it should be moved to the top level of the project (because individual project from the developer perspective should have no knowledge of the operator).